### PR TITLE
Changed syntax highlighting from JavaScript to es6

### PR DESCRIPTION
**Pull request by mjkaufer**
Also added a code block around a stray `eval()`

### CODE REVIEW

**Comment by justjake** [comment]
Dang, does Github's Javascript highlighter not work on these examples? I would prefer to keep `javascript` if it works fine because this code will eventually be normal javascript, but if this makes it pretty, then I'm in.

**Comment by ljharb** [comment]
It doesn't appear to make a difference to me:
```es6
class Foo { }
const { a } = { a: b => c };
```

```javascript
class Foo { }
const { a } = { a: b => c };
```

**Comment by hshoff** [comment]
This should be fixed from a while ago: https://github.com/github/linguist/issues/2398

File: README.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ Other Style Guides
       return `How are you, ${name}?`;
     }
     ```
-  - [6.5](#6.5) <a name='6.5'></a> Never use eval() on a string, it opens too many vulnerabilities.
+  - [6.5](#6.5) <a name='6.5'></a> Never use `eval()` on a string, it opens too many vulnerabilities.
```
Comment (ljharb) [inline README.md]: this is the only change in this PR that i think should stay

Comment (hshoff) [inline README.md]: Yup! fixed here: https://github.com/airbnb/javascript/commit/f130c0ffa26ae24f2e18bac33bc5cf7f62a2d938

**Pull request closed without merging**